frontend/src/views/Game.tsx

```diff
@@ -1,31 +1,68 @@
 import Editor from '@monaco-editor/react';
 import React, { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { Problem, getProblems } from '../api/Problem';
 import { Room } from '../api/Room';
+import { Text, ProblemHeaderText } from '../components/core/Text';
+import Header from '../components/navigation/Header';
 
 type LocationState = {
   room: Room,
 }
 
-function GamePage() {
+const FlexContainer = styled.div`
```

Comment:
Forgot to add this during the review, so leaving it as a single comment -- could these Container components go in the `Container.tsx` `core` file?